timemachine/ff/handlers/openmm_deserializer.py

```diff
@@ -200,8 +204,17 @@ def deserialize_system(system: mm.System, cutoff: float) -> Tuple[List[potential
             # use the same scale factors for electrostatics and lj
             scale_factors = np.stack([scale_factors_, scale_factors_], axis=1)
 
-            bps.append(potentials.Nonbonded(N, exclusion_idxs, scale_factors, beta, cutoff).bind(nb_params))
+            bps_dict["Nonbonded"].append(
+                potentials.Nonbonded(N, exclusion_idxs, scale_factors, beta, cutoff).bind(nb_params)
+            )
 
             # nrg_fns.append(('Exclusions', (exclusion_idxs, scale_factors, es_scale_factors)))
 
+    # ugh, ... various parts of our code assume the bps are in a certain order
+    # so put them back in that order here
+    bps = []
```

Comment:
What changed the ordering? Did a data structure change its ordering between python 3.7 and 3.11?

Reply:
Oh its probably the openmm update??

Reply:
I think this makes sense for this PR. Longer term, seems like we might want to use a dataclass instead of a list?

Reply:
Yeah, it's the openmm update. I agree, a better data structure could be used in the future.